packages/elements/src/internals/machines/sign-in/verification.machine.ts

```diff
@@ -92,49 +92,66 @@ const SignInVerificationMachine = setup({
+              ', ',
+            )}. Make sure to render a <Strategy> component for each supported strategy. More information: https://clerk.com/docs/elements/reference/sign-in#strategy`,
+          );
+        }
 
-      if (
-        clerk.client.signIn.supportedSecondFactors &&
-        !clerk.client.signIn.supportedSecondFactors.every(factor =>
-          context.registeredStrategies.has(factor.strategy as unknown as SignInFactor),
-        )
-      ) {
-        console.warn(
-          `Clerk: Your instance is configured to support these 2FA strategies: ${[
-            ...clerk.client.signIn.supportedSecondFactors,
-          ]
-            .map(f => f.strategy)
-            .join(', ')}, but the rendered strategies are: ${Array.from(context.registeredStrategies).join(
-            ', ',
-          )}. Before deploying your app, make sure to render a <Strategy> component for each supported strategy. For more information, visit the documentation: https://clerk.com/docs/elements/reference/sign-in#strategy`,
-        );
-      }
+        if (
+          clerk.client.signIn.supportedSecondFactors &&
+          !clerk.client.signIn.supportedSecondFactors.every(factor =>
+            context.registeredStrategies.has(factor.strategy as unknown as SignInFactor),
+          )
+        ) {
+          console.warn(
+            `Clerk: Your instance is configured to support these 2FA strategies: ${clerk.client.signIn.supportedSecondFactors
+              .map(f => f.strategy)
+              .join(', ')}, but the rendered strategies are: ${Array.from(context.registeredStrategies).join(
+              ', ',
+            )}. Make sure to render a <Strategy> component for each supported strategy. More information: https://clerk.com/docs/elements/reference/sign-in#strategy`,
+          );
+        }
+
+        // TODO: These "as SignInFactor" assertions are incorrect, factor.strategy is SignInFactor['strategy']. This needs to be fixed together with SignInVerificationStrategyRegisterEvent and SignInStrategy React component
+        // This type should also probably be SignInFirstFactor['strategy'] instead of SignInFactor['strategy'] !!!
+        const strategiesUsedButNotActivated = Array.from(context.registeredStrategies).filter(
+          strategy =>
+            !clerk.client.signIn.supportedFirstFactors.some(
+              supported => (supported.strategy as unknown as SignInFactor) === strategy,
+            ),
+        ) as unknown as Array<SignInFactor['strategy']>;
+
+        if (strategiesUsedButNotActivated.length > 0) {
+          console.warn(
+            `Clerk: These rendered strategies are not configured for your instance: ${strategiesUsedButNotActivated.join(', ')}. If this is unexpected, make sure to enable them in your Clerk dashboard: https://dashboard.clerk.com/last-active?path=/user-authentication/email-phone-username`,
+          );
+        }
```

Comment:
This is the new behavior. I think a diff is a nice DX touch we could in theory also use for the other warnings. Then you don't have to compare two lists